lib/helpers/fields/data-type-utils.js

```diff
@@ -0,0 +1,25 @@
+const {
+  isValidISODateString,
+  isValidDate
+} = require('iso-datestring-validator')
+
+function getDataTypeFromValue (value) {
+  if (typeof value === 'number') {
+    return Number.isInteger(value) ? 'Integer' : 'Double'
+  }
```

Comment:
Does this have to worry about catching any number that's an object `new Number()` instead of a primitive?

Reply:
possible I guess.